src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/dbscripts/querylogs.sql

```diff
@@ -141,11 +141,8 @@ SELECT
   "VHPhysIOKB",
   "WarningOnly",
   "WDName"
-FROM "{{baseDatabase}}"."{{#if vars.tableName}}{{vars.tableName}}{{else}}QryLogV{{/if}}"
-{{#if queryLogsVariables.timeRange}}
-WHERE "{{baseDatabase}}"."{{#if vars.tableName}}{{vars.tableName}}{{else}}QryLogV{{/if}}"."StartTime"
-  BETWEEN TIMESTAMP '{{queryLogsVariables.timeRange.startTimestamp}}' AND TIMESTAMP '{{queryLogsVariables.timeRange.endTimestamp}}'
-{{/if}}
+FROM "{{baseDatabase}}"."{{#if vars.tableName}}{{vars.tableName}}{{else}}QryLogV{{/if}}" AS "QLV"
```

Comment:
I like the helper implementation, but it has implicit dependency of "QLV". It should be better passed as an explicit parameter.
It's hard to tell if java implementation supports multi argument helpers, but it may be solved like that:

```
// template
{{#whereClauseForQuerylogs QLV}}{{/whereClauseForQuerylogs}}

// helper
SqlScriptVariables variables = (SqlScriptVariables) options.context.model(); 
QueryLogsVariables queryLogsVariables = variables.getQueryLogsVariables();
```

Reply:
Makes sense, and it turns out to be easier than that: options.param(0) is good enough.